README.md

```diff
@@ -2253,62 +2236,96 @@ Here is the table listing the default logging levels that come with the `logging
 
 For example, setting the logging level to `logging.DEBUG` will cause all the logging messages that are equal or higher than the `logging.DEBUG` in terms of severity to be emitted by your logger.
 
-Below is an example of this configuration.
+By default, logging level is set to `logging.INFO`.
 
-```python
-logging.basicConfig(level=logging.INFO)
-```
+To turn off the logging, you can set the `ClientConfig.logger_config.level` to a value higher than the numeric value of `logging.CRITICAL`. For example, the configuration below turns off the logging for the Hazelcast Python client.
 
-#### Setting Custom Handlers
+```python
+config.logger_config.level = 100  # Anything above the 50 will turn off the logging
+client = hazelcast.HazelcastClient(config)
+``` 
 
-Apart from `FileHandler` and `StreamHandler`, custom handlers can be added to the root logger using the `handlers` attribute.
+#### Setting a Custom Logging Configuration
 
-If specified, `handlers` should be the iterable of already created handlers. Any handler which don't already have a formatter set will be assigned to the default formatter created in the `logging.basicConfig`.
+`ClientConfig.logger_config.config_file` can be used to configure the logger for the Hazelcast Python client entirely.
+ 
+When set, this field should contain the absolute path of the JSON file that contains the logging configuration as described in the [Configuration dictionary schema](https://docs.python.org/3/library/logging.config.html#logging-config-dictschema). This file will be read and contents of it will be directly fed into the `logging.dictConfig` function.
```

Comment:
... read and the contents ...

(also you can use the simple present tense as a better structure)